.github/workflows/scripts/.pinot_compatibility_verifier.sh

```diff
@@ -45,6 +45,6 @@ echo "</settings>">> ${SETTINGS_FILE}
 
 # PINOT_MAVEN_OPTS is used to provide additional maven options to the checkoutAndBuild.sh command
 export PINOT_MAVEN_OPTS="-s $(pwd)/${SETTINGS_FILE}"
-compatibility-verifier/checkoutAndBuild.sh -w $WORKING_DIR -o $OLD_COMMIT
+compatibility-verifier/checkoutAndBuild.sh -w $WORKING_DIR -o $OLD_COMMIT -n $NEW_COMMIT
```

Comment:
I wonder how this works if new commit is null. the command should error out, since there i a `-n` option, but no commit id given.

Reply:
Maybe we should check if $NEW_COMMIT is specified before adding the `-n` option?

Reply:
Yes, we should, but I am wondering how the check passed in this PR

Reply:
If `NEW_COMMIT` is null, we pick the master's most recent PR.

Reply:
good to add a check for null of NEW_COMMIT. It is working because of a bug in `checkoutAndBuild.sh`